evap/staff/tests/test_views.py

```diff
@@ -2454,13 +2594,17 @@ def test_review_actions(self):
         let_user_vote_for_evaluation(self.app, self.student2, self.evaluation)
 
         # now reviewing should work
+        helper_enter_staff_mode(self)
```

Comment:
this seems wrong, there was no exit after setUp

Reply:
I fixed it (with a context manager)